src/main/java/org/apache/accumulo/testing/continuous/ManySplits.java

```diff
@@ -0,0 +1,266 @@
+                log.info(
+                    "{} tablets have file sizes not yet <= {} on table {}. Diff of avg offending file(s): {}",
+                    offendingTabletSizes.size(), splitThresholdStr, tableName,
+                    bytesToMemoryString(diff));
+              }
+              MILLISECONDS.sleep(sleepMillis);
+            }
+          } catch (Exception e) {
+            throw new RuntimeException(e);
+          }
+        });
+
+        long timeTaken = System.nanoTime() - beforeThresholdUpdate;
+
+        log.info(
+            "Time taken for all tables to reach expected total file size ({}): {} seconds ({}ms)",
+            splitThresholdStr, NANOSECONDS.toSeconds(timeTaken), NANOSECONDS.toMillis(timeTaken));
+      }
+
+      log.info("Test completed successfully.");
+
+      log.info("Deleting tables");
+      tableNames.stream().parallel().forEach(tableName -> {
+        try {
+          client.tableOperations().delete(tableName);
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
+      });
+      log.info("Deleting namespace");
+      client.namespaceOperations().delete(NAMESPACE);
+
+    }
+  }
+
+  /**
+   * @return a map of tablets to the sum of their file size
+   */
+  private static Map<Text,Long> getTabletFileSizes(AccumuloClient client, String tableName)
+      throws TableNotFoundException, AccumuloException, AccumuloSecurityException {
+    TableId tableId = TableId.of(client.tableOperations().tableIdMap().get(tableName));
+    try (Scanner scanner = client.createScanner("accumulo.metadata")) {
+      scanner.fetchColumnFamily("file");
+      scanner.setRange(createMetaRangeForDefaultTablet(tableId.canonical()));
+
+      Map<Text,Long> result = new HashMap<>();
+      for (var entry : scanner) {
+        String encodedDFV = new String(entry.getValue().get(), UTF_8);
+        String[] ba = encodedDFV.split(",", 2);
+        long tabletFileSize = Long.parseLong(ba[0]);
+        result.merge(entry.getKey().getRow(), tabletFileSize, Long::sum);
+      }
+
+      return result;
+    }
+  }
+
+  public static String bytesToMemoryString(long bytes) {
+    if (bytes < 1024) {
+      return bytes + "B"; // Bytes
+    } else if (bytes < 1024 * 1024) {
+      return (bytes / 1024) + "K"; // Kilobytes
+    } else if (bytes < 1024 * 1024 * 1024) {
+      return (bytes / (1024 * 1024)) + "M"; // Megabytes
+    } else {
+      return (bytes / (1024 * 1024 * 1024)) + "G"; // Gigabytes
+    }
+  }
+
+  private static Range createMetaRangeForDefaultTablet(String tableId) {
+    Text startRow = encodeRowForDefaultTablet(tableId);
+    Text endRow = new Text(startRow);
+    endRow.append(new byte[] {0}, 0, 1);
+
+    return new Range(startRow, true, endRow, false);
+  }
+
+  private static Text encodeRowForDefaultTablet(String tableId) {
+    Text entry = new Text(tableId);
+    entry.append(new byte[] {'<'}, 0, 1); // Delimiter for default tablet
+    return entry;
+  }
```

Comment:
I think the purpose of this code is to get the meta range for table, if so not sure it is.   Could do the following.


```suggestion
  private static Range getMetaRangeForTable(String tableId){
     return new Range(tableId+";", false, tableId+"<", true);
  }
```

Reply:
Fixed in 9fe07c0